code/modules/core_implant/cruciform/rituals/path.dm

```diff
@@ -250,6 +250,44 @@
 	phrase = "Dominus, ait, in cujus conspectu ambulo, mittet angelum suum tecum, et diriget viam tuam." //"He replied, ‘The Lord, before whom I have walked faithfully, will send his angel with you and make your journey a success.'"
 	stats_to_boost = list(STAT_ROB = 15, STAT_TGH = 15, STAT_VIG = 15)
 
+/datum/ritual/cruciform/lemniscate/holy_boost
+	name = "Bolster the Fellowship"
+	phrase = "Benedicat tibi Dominus, et custodiat te. Ostendat Dominus faciem suam tibi, et misereatur tui. Convertat Dominus vultum suum ad te, et det tibi pacem." //“The Lord bless you and keep you; the Lord make his face shine on you and be gracious to you; the Lord turn his face toward you and give you peace.”
+	var/stats_to_boost = list(STAT_MEC = 15, STAT_COG = 15, STAT_BIO = 15, STAT_ROB = 15, STAT_TGH = 15, STAT_VIG = 15)
+	desc = "This litany boosts all the stats of all disciples who hear you for thirty minutes, but has no effect on those without cruciforms."
+
+/datum/ritual/cruciform/lemniscate/holy_boost/perform(mob/living/carbon/human/user, obj/item/implant/core_implant/cruciform/C)
+	var/list/people_around = list()
+	for(var/mob/living/carbon/human/H in view(user))
+		if(H != user && !isdeaf(H) && is_neotheology_disciple(H)) //This one only for the homies
```

Comment:
```suggestion
		if(H != user && is_neotheology_disciple(H)) //This one only for the homies
```

Reply:
Accually ive looked at other prayers and this is a constant trend

Reply:
Because that's how the other ones work. It's not just "the cruciform hears it", it's words of inspiration. It could be changed, I'm just not sure it needs to.

Reply:
Ye i didnt know it was like that for the other spells i thought this was uniquic to this spell.
Its not constant internally tho, why can you pray well deaf and get the affect????